### PR TITLE
Prepare release 4.2.0 with changelog.

### PR DESCRIPTION
**Pull request by stephenashank**

### CODE REVIEW

**Comment by craigdbarber** [comment]
It looks like this needs to be re-based. Some commits for the permissions checking PR are showing up.

**Comment by stephenashank** [comment]
Those commits weren't added to this repository before this PR. I rebased the repository with these changes onto master before making this PR

**Comment by craigdbarber** [comment]
Gotcha. Thanks for checking :)

File: src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudOneShotInstanceIT.java [hunk 1]
```diff
@@ -99,7 +103,7 @@ public static void init() throws Exception {
     Builder otherStep = execute(Commands.ECHO, "\"also works\"");
     otherProject.getBuildersList().add(otherStep);
     otherProject.setAssignedLabel(new LabelAtom(LABEL));
-    otherBuildFuture = otherProject.scheduleBuild2(0);
+    otherBuildFuture = otherProject.scheduleBuild2(30);
```
Comment (craigdbarber) [inline src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudOneShotInstanceIT.java]: Suggest using a constant here, QUIET_PERIOD_SECS, or else leaving an in-line comment explaining what this literal represents.

**Pull request merged**